lib/home/surveys/take_survey/survey_theme.dart

```diff
@@ -47,7 +47,7 @@ class SurveyStyling extends StatelessWidget {
       primary: SurveyColors.veridian,
       primaryContainer: isLight ? SurveyColors.veridian : SurveyColors.orangeWhite,
       onPrimary: blackAndWhite,
-      secondary: SurveyColors.maroon,
+      secondary: isLight ? SurveyColors.darkGray : SurveyColors.maroon,
```

Comment:
The veridian/yellow colors that I previously chose for the yes/no button remind me of ketchup & mustard. 👎